config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/templates/ccdb-migrate-job.yaml

```diff
@@ -0,0 +1,26 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ccdb-migrate
+  namespace: #@ data.values.system_namespace
```

Comment:
we probably should add `kapp.k14s.io/update-strategy: fallback-on-replace` [1] to make sure we are always able to "update" the job. jobs in k8s are mostly immutable (in terms of field changes) so this would allow kapp to fallback on delete+create strategy.

[1] https://github.com/k14s/kapp/blob/master/docs/apply.md#controlling-apply-via-resource-annotations

Reply:
is there any concerns that this will run in parallel to capi deployment updates?

Reply:
4d461bdb050464d93a0252b519210b4f42322c8f Resolves this.